docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+  adheres to the `Helm version skew policy <https://helm.sh/docs/topics/version_skew/>`_.
+
+Running Trino using Helm
+------------------------
+
+Run the following commands from the system with ``helm`` and ``kubectl``
+installed and configured to connect to your running Kubernetes cluster:
+
+#. Validate kubectl is pointing to the correct cluster by running the command:
+
+   .. code-block:: text
+
+      kubectl cluster-info
+
+   You should see output that shows the correct Kubernetes control plane
+   address.
+
+#. Add the Trino Helm chart repository to Helm if you haven't done so already.
+   This tells Helm where to find the Trino charts. You can name the repository
+   whatever you want, ``trino`` is a good choice.
+
+   .. code-block:: text
+
+       helm repo add trino https://trinodb.github.io/charts
+
+#. Install Trino on the Kubernetes cluster using the Helm chart. Start by
+   running the ``install`` command to use all default values and create
+   a cluster called ``my-trino-cluster``.
+
+   .. code-block:: text
+
+       helm install my-trino-cluster trino/trino
+
+   This generates the Kubernetes configuration files by inserting properties
+   into helm templates. The helm chart contains
+   `default values <https://trinodb.github.io/charts/charts/trino/>`_
+   that can be overridden by a ``values.yaml`` file to update default settings.
+
+   * *(Optional)* To override the default values,
+     :ref:`create your own values.yaml <creating-your-own-values-yaml>` to
+     define the parameters of your deployment. To run the install command using
+     the ``values.yaml``, add the ``f`` parameter in you ``install`` command.
+
+     .. code-block:: text
+
+         helm install -f values.yaml my-trino-cluster trino/trino
+
+   You should see output as follows:
+
+   .. code-block:: text
+
+       NAME: my-trino-cluster
+       LAST DEPLOYED: Tue Sep 13 14:12:09 2022
+       NAMESPACE: default
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output depends on your ``values.yaml`` configuration and cluster name.
+   For example, the port ``8080`` is set by the ``.service.port`` in the
+   ``values.yaml``.
+
+#. Run the following command to check that all pods, deployments, and services
+   are running properly.
+
+   .. code-block:: text
+
+      kubectl get all
+
+   You should expect to see output that shows running pods, deployments, and
+   replica sets. A good indicator that everything is running properly is to see
+   all pods are returning a ready status in the  ``READY`` column.
```

Comment:
you mean a "Running" status in the STATUS column

or a 

1/1 in the READY column